app/src/main/java/com/yeonae/chamelezone/view/place/PlaceDetailActivity.kt

```diff
@@ -17,19 +17,6 @@ class PlaceDetailActivity: AppCompatActivity() {
         val imageAdapter = ImageViewPagerAdapter()
```

Comment:
class 멤버변수로 만들거나 
아니면
```suggestion
        view.adapter = ImageViewPagerAdapter()
```
이렇게 하거나 하는게 좋지 않을까요?

Reply:
수정했습니다